### ISSUE TITLE
Question: Can we do nested NER?

### ISSUE THREAD

**Issue by hkvision** [NervanaSystems/nlp-architect#37]
Can the NER module do the following?
tag `A B C` where A is an entity and ABC on the whole is another entity, a little bit like multi-label classification.

**Comment by peteriz** [comment]
Hi @hkvision, currently the NER model support 1 label set.
If you're interested you can implement multi-label tagging (adding support in the data loader and model)

**Comment by hkvision** [comment]
OK. Thanks. Kindly ask is there any reference for multi-label tagging?

**Comment by peteriz** [comment]
I'm not familiar with works in this area but perhaps [this](https://www.depends-on-the-definition.com/guide-to-multi-label-classification-with-neural-networks/) would help.

Intuitively, I can think of the following: you can define multi-class labels (encoded using 1/2/3-hot encoding) and cross entropy loss as a loss function. Finally, you can take the top-k predictions of the network to be the predicted classes.
